### DISCUSSION TITLE
How to use USD in filament

### DISCUSSION THREAD

**Discussion by AhrenLi** [google/filament#7064] [General]
Hi,
filament is a very fast and easy to use engine. I see a lot of examples showing glTf models that work great. But I don't know how to import USD type resources into filament. I found some conversion schemes, such as [usd_from_gltf](https://github.com/google/usd_from_gltf), [usd2gltf](https://github.com/mikelyndon/usd2gltf), etc., but the effect is not satisfactory. I would like to ask how to use USD in filament, thank you.

**Comment by romainguy** [comment]
You would have to write your own parser or start from a library like TinyUSD.
